src/sentry/static/sentry/app/views/admin/options.tsx

```diff
@@ -174,7 +174,13 @@ const definitions: Field[] = [
   },
   {
     key: 'mail.use-tls',
-    label: t('Use TLS?'),
+    label: t('Use STARTTLS? (exclusive with SSL)'),
+    component: BooleanField,
+    defaultValue: () => false,
+  },
+  {
+    key: 'mail.use-ssl',
+    label: t('Use SSL? (exclusive with STARTTLS)'),
```

Comment:
I wonder if we can find a way to make these radio buttons: `Insecure`, `STARTTLS`, `SSL`.

@EvanPurkhiser @billyvg @priscilawebdev ideas?

Reply:
It's doable, but I don't think there's a straightforward way to do it

Reply:
@PMExtra - would you like to at least add some logic check for when both of these are checked or enabled?